constellations/clients/orionFakeDataFiller/filler.go

```diff
@@ -228,28 +238,123 @@ func runFiller(hostAddress string) {
 	)
 
 	// Create sessions
-	createSession(
+	createSessions(
 		hostAddress,
-		"id_1",
+		"ap_java_2020_fall_class1",
 		"false",
+		3,
 	)
 
-	createSession(
+	createSessions(
 		hostAddress,
-		"id_2",
+		"ap_calculus_2020_fall_class1",
 		"true",
+		2,
+	)
+
+	// Create Accounts
+	createAccount(
+		hostAddress,
+		"emailaddress1@gmail.com",
```

Comment:
i think to make it more clear who is the owner of this account, make this email more specific. i.e. joesmith@gmail.com. And the user's email would need to change as well